Sources/ImageComposer.swift

```diff
@@ -66,9 +66,26 @@ final class ImageComposer {
         return context
     }
 
-    private func fillBackground(with color: NSColor, context: CGContext) {
-        context.setFillColor(color.cgColor)
-        context.fill(CGRect(x: 0, y: 0, width: context.width, height: context.height))
+    private func addBackground(_ background: Background, context: CGContext) {
+        switch background {
+        case .color(let color):
+            context.setFillColor(color.cgColor)
+            context.fill(CGRect(x: 0, y: 0, width: context.width, height: context.height))
+        case .linearGradient(let direction, let colors):
+            var locations = [CGFloat]()
+            locations.append(CGFloat(0))
```

Comment:
Nit: Could use the array building shorthand instead of appending a mutable array.